src/utils/use-previous.ts

```diff
@@ -1,5 +1,9 @@
 import {useEffect, useRef} from 'react';
 
+/**
+ * Returns a current reference to the previous value. The returned value is safe
+ * to use in dependency arrays as refs won't trigger reruns.
+ */
```

Comment:
Good to have comments like this imo :+1:

Reply:
Agreed. Easy to skip, but jsdoc/tsdoc like comments (as included on `usePollableResource`) is very handy to understand context for parameters and intention of function as well. So could even add parameters here and description of return value 😄